### PR TITLE
LPS-197350 The user cannot add or edit entries when the layout has the status field

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
[LPS-197350](https://liferay.atlassian.net/browse/LPS-197350)

### CODE REVIEW

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Review by mateussgn — APPROVED** [review]

**Comment by mateussgn** [comment]
Hey @MarinhoFeliphe, could you please double-check this PR?

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/form.jsp [hunk 1]
```diff
@@ -110,6 +110,10 @@ portletDisplay.setURLBack(backURL);
 					}
 				}
 
+				if (fieldName === 'status') {
+					value = null;
+				}
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/form.jsp:115]: Hi @pedro-oliveira446 please do this way at the beginning of the reduce method:

```javascript
				if (field.readOnly) {
					return obj;
				}
```

This way we can avoid magic names like 'status' and we're keeping this logic simple.

Thank you!

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Review by MarinhoFeliphe — APPROVED** [review]
LGTM!! :tada:

**Comment by pedro-oliveira446** [comment]
ci:forward:force

**Pull request closed without merging**